### PR TITLE
Add an email to teams we're putting on waiting list for SR2025

### PR DESCRIPTION
**Pull request by PeterJCLaw**
## Summary

We don't have places for everyone unfortunately. This email is pretty much a copy/paste of the same email from last year.

Fixes https://github.com/srobo/tasks/issues/1466

## Checklist

- [x] Has front-matter (`to` and `subject`)
- [x] Compared to a similar email from the previous year -- this is largely a copy/paste
- [x] Ensured there's enough context for a rookie team to understand

### CODE REVIEW

File: SR2025/2024-09-16-place-rejection.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+this year.
+
+We realise this is disappointing (we're disappointed too!), however we are
+limited by the number of kits we have and by the size of the venue. We are
+always looking for ways to support more teams and hope to be able to do so in
+future.
+
+We've put you on our waiting list so if we have any early drop outs we will be
+sure to let you know. You may be able to take their place, but we cannot promise
+anything. Let us know if you don't want to be on the waiting list.
```
Comment (samjmartin04) [inline SR2025/2024-09-16-place-rejection.md:19]: Do we want to say these to all teams that didn't make it? I don't know how many rejects there are, but I don't think it's worth having more than a few teams on the waiting list?

Comment (PeterJCLaw) [reply SR2025/2024-09-16-place-rejection.md:19]: I think this is what we've done in the past. We don't know if teams who are on the waiting list will actually want the place should one appear, so it's probably better to have a few more than we need.

If we have a deadline after which we won't reallocate a place I think that could also be informative around this. Let's discuss both aspects on Wednesday (I see the reallocation process is already on the agenda); I don't think this is urgent to send.

Comment (sedders123) [reply SR2025/2024-09-16-place-rejection.md:19]: We're not putting all teams on the waiting list, we are rejecting some teams. I think that means this is ok?

Comment (PeterJCLaw) [reply SR2025/2024-09-16-place-rejection.md:19]: Given we're doing two separate emails, yeah, let's keep this here and the other email can be different.

**Review by sedders123 — APPROVED** [review]

File: SR2025/2024-09-16-place-rejection.md (hunk 1 above)
Comment (sedders123) [inline SR2025/2024-09-16-place-rejection.md:19]: Making this conditional means we can send to both rejected and waiting listed teams
```suggestion
{{ if team_is_on_waiting_list }}
We've put you on our waiting list so if we have any early drop outs we will be
sure to let you know. You may be able to take their place, but we cannot promise
anything. Let us know if you don't want to be on the waiting list.
{{ end_if }}
```

**Review by sedders123 — APPROVED** [review]

**Comment by PeterJCLaw** [comment]
All sent. Have updated the [sign-ups sheet](https://docs.google.com/spreadsheets/d/1-jMpUNM4JLLYlNNpkjKg1mo44Yx1_a5gIoIV67OkL3M/edit) with colours to match -- grey for rejections, yellow for waiting list (the shades of green are the batches of place confirmations).

**Pull request merged**